include/ocpp/v16/charge_point.hpp

```diff
@@ -573,6 +573,12 @@ class ChargePoint {
         const std::function<DataTransferResponse(const RunningCost& session_cost, const uint32_t number_of_decimals)>&
             session_cost_callback);
 
+    /// \brief Registers a callback function for the session cost text message (California Pricing Requirements).
```

Comment:
I think it would help to give a little bit more context here and to describe whats expected from the callback.